### PR TITLE
Small changes + Added support for dungeonslice

### PR DESCRIPTION
**Pull request by Krealle**

### CODE REVIEW

**Comment by Saeldur** [comment]
How's these changes look ?
Can always leave pr's set to Draft if you don't want me prodding and perusing until they're "done"

File: apl/ptr.simc [hunk 1]
```diff
@@ -80,51 +79,53 @@ actions.aoe+=/living_flame,if=talent.snapfire&buff.burnout.up
...
 # Spend the last 1 or 2 GCDs of DR on fillers to exit with 2 EBs
 actions.st+=/living_flame,if=buff.dragonrage.up&buff.dragonrage.remains<(buff.essence_burst.max_stack-buff.essence_burst.stack)*gcd.max&buff.burnout.up
 actions.st+=/azure_strike,if=buff.dragonrage.up&buff.dragonrage.remains<(buff.essence_burst.max_stack-buff.essence_burst.stack)*gcd.max
 # Spend burnout procs without overcapping resources
-actions.st+=/living_flame,if=buff.burnout.up&(buff.leaping_flames.up&!buff.essence_burst.up|!buff.leaping_flames.up&buff.essence_burst.stack<buff.essence_burst.max_stack)&essence<essence.max-1 
-# In DR or if overflowing chain after the 3rd damage tick. Clip after third tick in dragonrage for more important buttons. Will only cast if >2s till movement
+actions.st+=/living_flame,if=buff.burnout.up&(buff.leaping_flames.up&!buff.essence_burst.up|!buff.leaping_flames.up&buff.essence_burst.stack<buff.essence_burst.max_stack)&essence<essence.max-1
+# In DR or if overflowing chain after the 3rd damage tick. Clip after fourth tick in dragonrage for more important buttons. Will only cast if >2s till movement
```
Comment (Saeldur) [inline apl/ptr.simc:117]: I missed when you made this change, how much was it from changing it from on third tick?

**Comment by Krealle** [comment]
> 

ES change: 
6t: https://www.raidbots.com/simbot/report/jLhmtC6dpAURKBpTg3ji3G
7t: https://www.raidbots.com/simbot/report/djeBJwfFtG9RCnBMQVzAT8

no vola change:
https://www.raidbots.com/simbot/report/dYFKoAD5amw9VgDpE9ABkn

**Comment by Krealle** [comment]
@Saeldur can you look this over and get it merged?

**Pull request merged**